kernel/aster-nix/src/fs/procfs/meminfo.rs

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: MPL-2.0
+
+//! This module offers `/proc/meminfo` file support, which tells the user space about the memory
+//! statistics in the entire system. The definition of the fields are similar to that of Linux's but there
+//! exists differences.
+//!
+//! Reference: <https://man7.org/linux/man-pages/man5/procfs.5.html>
```

Comment:
I discover that https://man7.org/linux/man-pages/man5/proc_meminfo.5.html is a better reference. XD